lib/ansible/modules/cloud/vmware/vmware_host_facts.py

```diff
@@ -322,6 +322,7 @@ def get_system_facts(self):
             'ansible_bios_version': self.host.hardware.biosInfo.biosVersion,
             'ansible_uptime': self.host.summary.quickStats.uptime,
             'ansible_in_maintenance_mode': self.host.runtime.inMaintenanceMode,
+            'ansible_uuid': self.host.hardware.systemInfo.uuid,
```

Comment:
Could you please add a note in documentation stating this fact is added in Ansible 2.10 version and also, an example of returned fact in `RETURN`? Thanks.